### PR TITLE
add updated dlm waiver

### PR DESCRIPTION
**Pull request by anthonygwalters**
# Pull Request

## Summary & Motivation

[//]: # "When merged, this pull request will..."

## Self-review

- [ ] Update **due date**, **assignee**, and **priority** on our
      [TEAMster Asana Project](https://app.asana.com/0/1205971774138578/1205971926225838)

  _If this is a same-day request, please flag that in Slack_

- [ ] <kbd>Format</kbd> has been run on all modified files

- [ ] Ensure you are using the `union_dataset_join_clause()` macro for queries that employ any
      models from these datasets:
  - deanslist
  - edplan
  - iready
  - pearson
  - powerschool
  - renlearn
  - titan

## Troubleshooting

- [SqlFluff Rules Reference](https://docs.sqlfluff.com/en/stable/rules.html)
- [Trunk](https://teamschools.github.io/teamster/CONTRIBUTING/#trunk)
- [dbt](https://teamschools.github.io/teamster/CONTRIBUTING/#dbt-cloud_1)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207934580285019

### CODE REVIEW

File: src/dbt/kipptaf/models/reporting/intermediate/int_reporting__student_filters.sql [hunk 1]
```diff
@@ -148,6 +148,55 @@ with
...
+                when co.grade_level >= 4 and nj.math_state_assessment_name = '3'
+                then true
+                else false
+            end as is_exempt,
+        from {{ ref("base_powerschool__student_enrollments") }} as co
+        left join
+            {{ ref("stg_powerschool__s_nj_stu_x") }} as nj
+            on co.students_dcid = nj.studentsdcid
+            and {{ union_dataset_join_clause(left_alias="co", right_alias="nj") }}
     )
```
Comment (cbini) [inline src/dbt/kipptaf/models/reporting/intermediate/int_reporting__student_filters.sql:200]: this should pull solely from `stg_powerschool__s_nj_stu_x` and then move the case statements to the main select, which is already pulling in `base_powerschool__student_enrollments`

**Review by cbini — CHANGES_REQUESTED** [review]

File: src/dbt/kipptaf/models/reporting/intermediate/int_reporting__student_filters.sql [hunk 2]
```diff
@@ -148,6 +148,55 @@ with
...
+                when co.grade_level >= 4 and nj.state_assessment_name in ('2', '3', '4')
+                then true
+                else false
+            end as is_exempt,
+        from {{ ref("base_powerschool__student_enrollments") }} as co
+        left join
+            {{ ref("stg_powerschool__s_nj_stu_x") }} as nj
+            on co.students_dcid = nj.studentsdcid
+            and {{ union_dataset_join_clause(left_alias="co", right_alias="nj") }}
+        where student_number = 100949
```
[the remark this answers is not in the document]
Comment (anthonygwalters) [reply src/dbt/kipptaf/models/reporting/intermediate/int_reporting__student_filters.sql]: removed

File: src/dbt/kipptaf/models/reporting/intermediate/int_reporting__student_filters.sql (hunk 1 above)
Comment (anthonygwalters) [reply src/dbt/kipptaf/models/reporting/intermediate/int_reporting__student_filters.sql]: changes made

**Comment by cbini** [comment]
rebasing this, Gaby's pull request has a useful int view this can use

**Pull request merged**